Lib/opcode.py

```diff
@@ -219,6 +205,45 @@ def jabs_op(name, op):
 
 del def_op, name_op, jrel_op, jabs_op
 
+_nb_slots = [
```

Comment:
The first element of each entry is redundant. `_nb_slots[i][0] == i`

Reply:
Those values correspond to offsets in `PyNumberMethods`, not indexes into this list. Many of the slots are omitted here, resulting in gaps.

`_nb_slots[i][0] != i` for all `i`.

Reply:
I was mislead by the comments. Could you remove the comments, and rename the list, as it isn't a list of the number slots, but a list of the binary operand arguments.